src/main/kotlin/blackjack/ui/ResultView.kt

```diff
@@ -0,0 +1,53 @@
+package blackjack.ui
+
+import blackjack.domain.PlayerWinType
+import blackjack.ui.model.PlayerCardResult
+import blackjack.ui.model.PlayerDto
+import blackjack.ui.model.PlayerWinTypes
+
+object ResultView {
+
+    fun printCardsOf(playerDtos: List<PlayerDto>) {
+        val names = playerDtos.joinToString(", ") { it.name }
+        println("${names}에게 2장의 나누었습니다.")
+        for (playerDTO in playerDtos) {
+            printCardsOfSinglePlayer(playerDTO)
+        }
+        println()
+    }
+
+    fun printCardsOfSinglePlayer(playerDto: PlayerDto) {
+        println("${playerDto.name}: ${playerDto.cards.joinToString(", ") { it.toString() }}")
+    }
+
+    fun printInfoOfThirdCardOfDealer() {
+        println("딜러는 16이하라 한장의 카드를 더 받았습니다.")
+        println()
+    }
+
+    fun printCardResults(playerCardResults: List<PlayerCardResult>) {
+        for (blackJackResult in playerCardResults) {
+            printSingleResult(blackJackResult)
+        }
+        println()
+    }
+
+    private fun printSingleResult(playerCardResult: PlayerCardResult) {
+        println("${playerCardResult.name}: ${playerCardResult.cardNames.joinToString()} - 결과: ${playerCardResult.point}")
+    }
+
+    fun printWinningResult(playerWinTypes: PlayerWinTypes) {
+        println("## 최종 승패")
+        println(playerWinTypes.dealerResult)
+        playerWinTypes.keys.forEach {
+            println("$it: ${playerWinTypes[it]?.description}")
+        }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        playerWinTypes.forEach {
            println("${it.key}: ${it.value.description}")
        }
```